modules/json/src/main/java/com/devonfw/module/json/common/base/ObjectMapperFactory.java

```diff
@@ -141,4 +147,17 @@ public ObjectMapper createInstance() {
 
     return mapper;
   }
+
+  /**
+   * @return an instance of {@link SimpleModule} for registering configurations
+   */
+  protected SimpleModule initMapping() {
```

Comment:
great. Just call it from the constructor of this ObjectMapperFactory.

Reply:
Done